src/msw/listctrl.cpp

```diff
@@ -3689,12 +3689,24 @@ static void wxConvertToMSWListCol(HWND hwndList,
         // seem to be generally nicer than on the left and the generic
         // version only draws them on the right (we don't have a flag to
         // specify the image location anyhow)
-        const int fmtImage = LVCFMT_BITMAP_ON_RIGHT | LVCFMT_COL_HAS_IMAGES;
-
-        if ( item.m_image != -1 )
+        const int fmtImage = LVCFMT_IMAGE | LVCFMT_BITMAP_ON_RIGHT; // Not sure if LVCFMT_COL_HAS_IMAGES means the whole column header is an image
+                                                                    // LVCFMT_IMAGE works well
+        if (item.m_image != -1)
+        {
+            lvCol.mask |= LVCF_IMAGE; // Since Windows Common Control Version 4.70,
```

Comment:
I don't think this is needed, it's already set above unconditionally anyhow.

Alternatively, we can keep this one, but remove the line setting it above and the one resetting it below. I think I actually prefer doing it like this...

Reply:
Sorry for my carelessness.
The way that set LVCF_IMAGE conditionally makes the code looks symmetrical :)